test/spec/modules/userId_spec.js

```diff
@@ -1314,6 +1357,8 @@ describe('User ID', function() {
             name: 'sharedId', storage: {name: 'sharedid', type: 'cookie'}
           }, {
             name: 'intentIqId', storage: { name: 'intentIqId', type: 'cookie' }
+          }, {
+            name: 'zeotapIdPlus', storage: { name: 'IDP', type: 'cookie' }
```

Comment:
per above, this can probably just be your name
```suggestion
            name: 'zeotapIdPlus'
```

Reply:
made changes.